torch/_meta_registrations.py

```diff
@@ -7399,6 +7399,192 @@ def _create_grouped_mm_output_tensor(mat1, mat2, offs, out_dtype):
     return out
 
 
+def _meta_scaled_grouped_mm_fp8(
```

Comment:
Note for reviewers: I refactored _scaled_grouped_mm meta registrations to have separate branches for (1) fp8 rowwise, (2a) mxfp8 2d-2d, and (2b) 2d-3d branches.